code/modules/mob/living/simple_animal/constructs/constructs.dm

```diff
@@ -233,6 +237,7 @@
 	resistance = 10
 	var/energy = 0
 	var/max_energy = 1000
+	can_climb = 1 //It flies, its magic
```

Comment:
Giving juggernauts the ability to climb is stupid and shouldn't be done, honestly. I agree with what was said above.